tests/users/signup.test.js

```diff
@@ -0,0 +1,83 @@
+import chaiHttp from "chai-http";
+import chai, { expect, should } from "chai";
+import app from "../../index";
+
+const userDamie = {
```

Comment:
@abayo-luc userDamie??? or dummyUser??? Please,  let use meaningful names !

Reply:
I got it!! it was making sense in my head, but I get what you are trying to mean.